### PR TITLE
chore: Optimizing hertz observability

### PR DESCRIPTION
**Pull request by rogerogers**
#### Check the PR title.

- [ ] This PR title match the format: \<type\>(optional scope): \<description\>
- [ ] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional): 优化 hertz 可观测性文档

#### (Optional) Which issue(s) this PR fixes:

Fixes https://github.com/hertz-contrib/obs-opentelemetry/issues/25

### CODE REVIEW

**Comment by GuangmingLuo** [comment]
@CoderPoet Please take a look.

File: content/zh/docs/hertz/tutorials/observability/instrumentation.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+1. `ReadHeaderStart`：读取 header 开始
+2. `ReadHeaderFinish`：读取 header 结束
+3. `ReadBodyStart`：读取 body 开始
+4. `ReadBodyFinish`：读取 body 结束
+5. `ServerHandleStart`：业务 handler 开始
+6. `ServerHandleFinish`：业务 handler 结束
+7. `WriteStart`：写 response 开始
+8. `WriteFinish`：写 response 结束
+
+> 如果不希望记录这些信息，可以不注册任何 tracer 或者将埋点策略设置为 `LevelDisabled`，则框架不会记录这些信息。
```
Comment (CoderPoet) [inline content/zh/docs/hertz/tutorials/observability/instrumentation.md:51]: 不注册 tracer 或者 LevelDisabled ，可能会导致 span/metrics 丢失，这块是否需要提示下？

Comment (rogerogers) [reply content/zh/docs/hertz/tutorials/observability/instrumentation.md:51]: 我加下

Comment (rogerogers) [reply content/zh/docs/hertz/tutorials/observability/instrumentation.md:51]: 已添加

**Review by li-jin-gou — APPROVED** [review]

**Pull request merged**